sdk/appcenter/src/androidTest/java/com/microsoft/appcenter/AppCenterAndroidTest.java

```diff
@@ -22,11 +22,7 @@
 import java.util.concurrent.Semaphore;
 import java.util.concurrent.atomic.AtomicReference;
 
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertNotEquals;
-import static org.junit.Assert.assertNotNull;
-import static org.junit.Assert.assertNull;
-import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.*;
```

Comment:
No star import, please always to the following with android commit dialog: organize imports, reformat, code analysis etc... Organize imports will automatically replace start with actual imports.

Reply:
This is a change came from the base branch

Reply:
I commented on that PR too on other places, but since it appears green (new changes), it also applies on yours.

Reply:
Looks like a github bug, the full diff does not show it.